public/Get-DbaBackupInformation.ps1

```diff
@@ -273,6 +274,24 @@ function Get-DbaBackupInformation {
                             Write-Message -Level VeryVerbose -Message "File"
                             $Files += $f.FullName
                         }
+                    } elseif ($f -match "^http") {
+                        $uri = [System.Uri]$f;
+                        $storageAccountName = $uri.Host.Split('.')[0];
+                        $pathSegments = $uri.AbsolutePath.Trim('/').Split('/');
+                        $containerName = $pathSegments[0];
+                        $prefix = if ($pathSegments.Length -gt 1) { $pathSegments[1] } else { "" }
+                        $sasToken = $uri.Query.TrimStart('?')
+                        $ctx = New-AzStorageContext -StorageAccountName $storageAccountName -SasToken "$sasToken";
```

Comment:
This was the only way I could find to obtain the blobs in a container through powershell, I really don't like the approach but it's the only way I could come up with.